koji-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora
+FROM registry.fedoraproject.org/fedora:33
```

Comment:
Some (maybe all?) `FROM` lines are replaced in the OpenShift build configs. You will need to update the base images there instead (and maybe add a comment about that in the Dockerfiles)

Reply:
(or you could remove the overrides as you have done for koji-db)

Reply:
Here's the thing, though... it actually **works** this way (registry authentication in particular), and doesn't work as it was before 😳

Reply:
Oh, I see :D
In that case, removing the overrides from the build configs may help avoid confusion

Reply:
Yeah, that was my motivation there :)

Reply:
I also found overrides in these files:
- openshift/templates/koji-base.yaml
- openshift/templates/cachito-worker.yaml
- openshift/templates/cachito-api.yaml

Wanna remove those as well?

Reply:
Agreed. At the time, I was thinking they were OK, as they're pulling from Fedora, but now I'm thinking it *is* better to have them in one place rather than both...